site2/docs/reference-pulsar-admin.md

```diff
@@ -2333,17 +2342,252 @@ Remove the message TTL for a topic.
 Usage
 ```bash
 $ pulsar-admin topics remove-message-ttl tenant/namespace/topic
-=======
+
+Options  
+```
+|Flag|Description|Default|
 |---|---|---|
 |`--enable`, `-e`|Enable message deduplication on the specified topic.|false|
 |`--disable`, `-d`|Disable message deduplication on the specified topic.|false|
 
-### `remove-deduplication`
-Remove a deduplication policy from a topic.
+### `get-retention`
+Get the retention policy that is applied to a topic.
 
 Usage
 ```bash
-$ pulsar-admin topics remove-deduplication tenant/namespace/topic
+$ pulsar-admin topics get-retention tenant/namespace/topic
+```
+
+### `set-retention`
+Set the retention policy that is applied to a topic.
+
+Usage
+```bash
+$ pulsar-admin topics set-retention tenant/namespace/topic
+```
+
+### `remove-retention`
+Remove the retention policy that is applied to a topic.
+
+Usage
+```bash
+$ pulsar-admin topics remove-retention tenant/namespace/topic
+```
+
+### `get-dispatch-rate`
+Get message-dispatch-rate for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics set-dispatch-rate tenant/namespace/topic
+```
+
+### `set-dispatch-rate`
+Set message-dispatch-rate for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics set-dispatch-rate tenant/namespace/topic
+```
+
+### `remove-dispatch-rate`
+Remove message-dispatch-rate for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics remove-dispatch-rate tenant/namespace/topic
+```
+
+### `get-delayed-delivery`
+Get the delayed delivery policy for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics get-delayed-delivery tenant/namespace/topic
+```
+
+### `set-delayed-delivery`
+Set the delayed delivery policy for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics set-delayed-delivery tenant/namespace/topic
+```
+
+### `remove-delayed-delivery`
+Remove the delayed delivery policy for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics remove-delayed-delivery tenant/namespace/topic
+```
+
+### `get-max-producers`
```

Comment:
can you please check if this is the correct command? according to https://github.com/apache/pulsar/blob/7c09f5ce649edcca0be792198d97573197c5a272/pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdTopics.java#L162-L164, the command should be `get-maxProducers` / `set-maxProducers` / `remove-maxProducers`

Reply:
@freeznet The functions were changed in this file pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdTopics.java Please take a look. Thank you.

Reply:
@sijia-w yes, but the change has been overwrite by #8003
https://github.com/apache/pulsar/pull/8003/files#diff-98564f4467b3071af3e1b5fc1d5a0703362d8d9487632cecc12c67d90abda4d2